webpack.config.client.js

```diff
@@ -73,12 +73,18 @@ module.exports = ({
         }),
       ],
       // specify min/max file sizes for each JS chunk for optimal performance
+      chunkIds: 'named',
       splitChunks: {
         chunks: 'initial',
         automaticNameDelimiter: '-',
         minSize: 184320, // 180kb
         maxSize: 245760, // 240kb
         cacheGroups: {
+          common: {
+            name: true,
```

Comment:
This gives a nice readable name to show which bundles the common elements have been extracted from: 
![image](https://user-images.githubusercontent.com/9645462/88309006-ab424900-cd05-11ea-82bc-5a58e692947a.png)
In production they recommend turning this off but it's great for debugging: https://webpack.js.org/plugins/split-chunks-plugin/#splitchunksname